src/pages/specification/includes/media-example.html

```diff
@@ -1,11 +1,11 @@
 <video src="myvideo" onmonetization="sayThanks(this)">
-  <link rel="monetization" href="https://example.com/video/pay" />
+  <link rel="monetization" href="https://example.com/video/pay">
 </video>
 
```

Comment:
One thing to check is the tidy configuration

Reply:
https://github.com/WICG/webmonetization/blob/main/src/pages/specification/tidyconfig.txt